build/web/common.cfg

```diff
@@ -0,0 +1,10 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+action {
+  define_artifacts {
+    regex: "github/filament/out/*.tgz"
+    regex: "github/filament/out/*.aar"
```

Comment:
Now that I think about it, the Windows build config has the same line, but we probably don't need to worry about `.aar` files for web builds

Reply:
It doesn't hurt to keep it. I left it in every build just in case.